![Fortune 500 Global Companies](https://s3.amazonaws.com/dq-content/291/fortune-500.jpg)

# Fortune 500 Companies 2017

## Description

**[Fortune 500 company list](https://fortune.com/fortune500/)** is a compilation of the largest companies in the United States as published by **[Fortune Magazine](https://fortune.com/)**

## Data set

The data set is a CSV file called `f500.csv`. Here is a data dictionary for some of the columns in the CSV:

* **`company`:** Name of the company.
* **`rank`:** Global 500 rank for the company.
* **`revenues`:** Company's total revenue for the fiscal year, in millions of dollars (USD).
* **`revenue_change`:** Percentage change in revenue between the current and prior fiscal year.
* **`profits`:** Net income for the fiscal year, in millions of dollars (USD).
* **`ceo`:** Company's Chief Executive Officer.
* **`industry`:** Industry in which the company operates.
* **`sector`:** Sector in which the company operates.
* **`previous_rank`:** Global 500 rank for the company for the prior year.
* **`country`:** Country in which the company is headquartered.

In [2]:
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
%matplotlib inline

In [7]:
f500 = pd.read_csv('../data/f500.csv', index_col=0)
f500.head()

rank  revenues  revenue_change  profits  assets  \
company                                                                     
Walmart                      1    485873             0.8  13643.0  198825   
State Grid                   2    315199            -4.4   9571.3  489838   
Sinopec Group                3    267518            -9.1   1257.9  310726   
China National Petroleum     4    262573           -12.3   1867.5  585619   
Toyota Motor                 5    254694             7.7  16899.3  437575   

                          profit_change                  ceo  \
company                                                        
Walmart                            -7.2  C. Douglas McMillon   
State Grid                         -6.2              Kou Wei   
Sinopec Group                     -65.0            Wang Yupu   
China National Petroleum          -73.7        Zhang Jianhua   
Toyota Motor                      -12.3          Akio Toyoda   

                                          industry                  sector  \
company                                                                      
Walmart                      General Merchandisers               Retailing   
State Grid                               Utilities                  Energy   
Sinopec Group                   Petroleum Refining                  Energy   
China National Petroleum        Petroleum Refining                  Energy   
Toyota Motor              Motor Vehicles and Parts  Motor Vehicles & Parts   

                          previous_rank country      hq_location  \
company                                                            
Walmart                               1     USA  Bentonville, AR   
State Grid                            2   China   Beijing, China   
Sinopec Group                         4   China   Beijing, China   
China National Petroleum              3   China   Beijing, China   
Toyota Motor                          8   Japan    Toyota, Japan   

                                               website  \
company                                                  
Walmart                         http://www.walmart.com   
State Grid                      http://www.sgcc.com.cn   
Sinopec Group                   http://www.sinopec.com   
China National Petroleum        http://www.cnpc.com.cn   
Toyota Motor              http://www.toyota-global.com   

                          years_on_global_500_list  employees  \
company                                                         
Walmart                                         23    2300000   
State Grid                                      17     926067   
Sinopec Group                                   19     713288   
China National Petroleum                        17    1512048   
Toyota Motor                                    23     364445   

                          total_stockholder_equity  
company                                             
Walmart                                      77798  
State Grid                                  209456  
Sinopec Group                               106523  
China National Petroleum                    301893  
Toyota Motor                                157210

In [6]:
f500.tail()

rank  revenues  revenue_change  profits  \
company                                                                   
Teva Pharmaceutical Industries   496     21903            11.5    329.0   
New China Life Insurance         497     21796           -13.3    743.9   
Wm. Morrison Supermarkets        498     21741           -11.3    406.4   
TUI                              499     21655            -5.5   1151.7   
AutoNation                       500     21609             3.6    430.5   

                                assets  profit_change                 ceo  \
company                                                                     
Teva Pharmaceutical Industries   92890          -79.3   Yitzhak Peterburg   
New China Life Insurance        100609          -45.6            Wan Feng   
Wm. Morrison Supermarkets        11630           20.4      David T. Potts   
TUI                              16247          195.5   Friedrich Joussen   
AutoNation                       10060           -2.7  Michael J. Jackson   

                                                       industry  \
company                                                           
Teva Pharmaceutical Industries                  Pharmaceuticals   
New China Life Insurance        Insurance: Life, Health (stock)   
Wm. Morrison Supermarkets                  Food and Drug Stores   
TUI                                             Travel Services   
AutoNation                                  Specialty Retailers   

                                            sector  previous_rank  country  \
company                                                                      
Teva Pharmaceutical Industries         Health Care              0   Israel   
New China Life Insurance                Financials            427    China   
Wm. Morrison Supermarkets       Food & Drug Stores            437  Britain   
TUI                              Business Services            467  Germany   
AutoNation                               Retailing              0      USA   

                                         hq_location  \
company                                                
Teva Pharmaceutical Industries  Petach Tikva, Israel   
New China Life Insurance              Beijing, China   
Wm. Morrison Supermarkets          Bradford, Britain   
TUI                                 Hanover, Germany   
AutoNation                       Fort Lauderdale, FL   

                                                    website  \
company                                                       
Teva Pharmaceutical Industries     http://www.tevapharm.com   
New China Life Insurance        http://www.newchinalife.com   
Wm. Morrison Supermarkets          http://www.morrisons.com   
TUI                                 http://www.tuigroup.com   
AutoNation                        http://www.autonation.com   

                                years_on_global_500_list  employees  \
company                                                               
Teva Pharmaceutical Industries                         1      56960   
New China Life Insurance                               2      54378   
Wm. Morrison Supermarkets                             13      77210   
TUI                                                   23      66779   
AutoNation                                            12      26000   

                                total_stockholder_equity  
company                                                   
Teva Pharmaceutical Industries                     33337  
New China Life Insurance                            8507  
Wm. Morrison Supermarkets                           5111  
TUI                                                 3006  
AutoNation                                          2310

In [8]:
f500.dtypes

rank                          int64
revenues                      int64
revenue_change              float64
profits                     float64
assets                        int64
profit_change               float64
ceo                          object
industry                     object
sector                       object
previous_rank                 int64
country                      object
hq_location                  object
website                      object
years_on_global_500_list      int64
employees                     int64
total_stockholder_equity      int64
dtype: object

information about the `f500.csv`

In [9]:
f500.info()

<class 'pandas.core.frame.DataFrame'>
Index: 500 entries, Walmart to AutoNation
Data columns (total 16 columns):
rank                        500 non-null int64
revenues                    500 non-null int64
revenue_change              498 non-null float64
profits                     499 non-null float64
assets                      500 non-null int64
profit_change               436 non-null float64
ceo                         500 non-null object
industry                    500 non-null object
sector                      500 non-null object
previous_rank               500 non-null int64
country                     500 non-null object
hq_location                 500 non-null object
website                     500 non-null object
years_on_global_500_list    500 non-null int64
employees                   500 non-null int64
total_stockholder_equity    500 non-null int64
dtypes: float64(3), int64(7), object(6)
memory usage: 66.4+ KB


In [11]:
f500.describe().T

count           mean            std      min  \
rank                      500.0     250.500000     144.481833      1.0   
revenues                  500.0   55416.358000   45725.478963  21609.0   
revenue_change            498.0       4.538353      28.549067    -67.3   
profits                   499.0    3055.203206    5171.981071 -13038.0   
assets                    500.0  243632.274000  485193.716285   3717.0   
profit_change             436.0      24.152752     437.509566   -793.7   
previous_rank             500.0     222.134000     146.941961      0.0   
years_on_global_500_list  500.0      15.036000       7.932752      1.0   
employees                 500.0  133998.316000  170087.825705    328.0   
total_stockholder_equity  500.0   30628.076000   43642.576833 -59909.0   

                                25%       50%         75%        max  
rank                        125.750    250.50     375.250      500.0  
revenues                  29003.000  40236.00   63926.750   485873.0  
revenue_change               -5.900      0.55       6.975      442.3  
profits                     556.950   1761.60    3954.000    45687.0  
assets                    36588.500  73261.50  180564.000  3473238.0  
profit_change               -22.775     -0.35      17.700     8909.5  
previous_rank                92.750    219.50     347.250      500.0  
years_on_global_500_list      7.000     17.00      23.000       23.0  
employees                 42932.500  92910.50  168917.250  2300000.0  
total_stockholder_equity   7553.750  15809.50   37828.500   301893.0

## Types of Industries

Below are the types of countries selected for the Fortune 500 and their total counts.

In [13]:
f500['industry'].value_counts()

Banks: Commercial and Savings                     51
Motor Vehicles and Parts                          34
Petroleum Refining                                28
Insurance: Life, Health (stock)                   24
Food and Drug Stores                              20
Mining, Crude-Oil Production                      18
Insurance: Property and Casualty (Stock)          18
Utilities                                         18
Telecommunications                                18
Trading                                           15
Pharmaceuticals                                   15
Aerospace and Defense                             14
Engineering, Construction                         13
Electronics, Electrical Equip.                    13
Metals                                            12
Energy                                            12
Specialty Retailers                               10
Industrial Machinery                              10
Insurance: Life, Health (Mutual)              

## Unique Countries from where the industries are located

Below are the countries and the number of companies in each country

In [14]:
f500['country'].value_counts()

USA             132
China           109
Japan            51
France           29
Germany          29
Britain          24
South Korea      15
Switzerland      14
Netherlands      14
Canada           11
Spain             9
Australia         7
Brazil            7
India             7
Italy             7
Taiwan            6
Russia            4
Ireland           4
Sweden            3
Singapore         3
Mexico            2
Norway            1
Denmark           1
Venezuela         1
Indonesia         1
Thailand          1
Saudi Arabia      1
Belgium           1
Finland           1
Turkey            1
U.A.E             1
Luxembourg        1
Israel            1
Malaysia          1
Name: country, dtype: int64

## Unique Companies Sector

In [15]:
f500['sector'].value_counts()

Financials                       118
Energy                            80
Technology                        44
Motor Vehicles & Parts            34
Wholesalers                       28
Health Care                       27
Food & Drug Stores                20
Transportation                    19
Telecommunications                18
Retailing                         17
Materials                         16
Food, Beverages & Tobacco         16
Industrials                       15
Aerospace & Defense               14
Engineering & Construction        13
Chemicals                          7
Media                              3
Business Services                  3
Hotels, Restaurants & Leisure      3
Household Products                 3
Apparel                            2
Name: sector, dtype: int64

## HQ Location

In [16]:
f500['hq_location'].value_counts()

Beijing, China              56
Tokyo, Japan                36
Paris, France               17
New York, NY                15
London, Britain             14
Seoul, South Korea          12
Shanghai, China              8
Osaka, Japan                 7
Zurich, Switzerland          7
Toronto, Ontario, Canada     7
Shenzhen, China              6
Hong Kong, China             6
Houston, TX                  5
Madrid, Spain                5
Amsterdam, Netherlands       5
Mumbai, India                5
Munich, Germany              4
Atlanta, GA                  4
Moscow, Russia               4
Chicago, IL                  4
Taipei, Taiwan               3
Courbevoie, France           3
Singapore                    3
Hanover, Germany             3
Sao Paulo, Brazil            3
Dublin, Ireland              3
Cincinnati, OH               3
Essen, Germany               3
Guangzhou, China             3
Hangzhou, China              3
                            ..
Kuala Lumpur, Malaysia       1
Taiyuan,

## rank change
to check the change from the previous rank to the current rank. `f500` dataframe includes each company's current and previous year's rank on the Fortune 500 list

In [17]:
rank_change = f500['previous_rank'] - f500['rank']
rank_change

company
Walmart                                           0
State Grid                                        0
Sinopec Group                                     1
China National Petroleum                         -1
Toyota Motor                                      3
Volkswagen                                        1
Royal Dutch Shell                                -2
Berkshire Hathaway                                3
Apple                                             0
Exxon Mobil                                      -4
McKesson                                          1
BP                                               -2
UnitedHealth Group                                4
CVS Health                                        4
Samsung Electronics                              -2
Glencore                                         -2
Daimler                                          -1
General Motors                                    2
AT&T                                              4
EXOR

We can observe from the results that `Sinopec Group` and `Toyota Motor` each increased in rank from the previous year, while China National Petroleum dropped a spot. what if we wanted to find the biggest increase or decrease in rank?

In [18]:
rank_change.max()

226

In [21]:
f500[['country', 'industry', 'sector']].describe()

country                       industry      sector
count      500                            500         500
unique      34                             58          21
top        USA  Banks: Commercial and Savings  Financials
freq       132                             51         118

**Findings:**

* `34`Countries, `58` industries, `21` sectors were featured in the 2017 Fortune 500.
* The `USA` is the country that headquarters the most Fortune 500 companies, with exactly `132` companies being featured. `Banks: Commercial and Savings` are the industry that were featured most in the Fortune 500 companies, `51` being featured. `Finacial` sector were featured most, `118` being featured.
